pkg/url/url_test.go

```diff
@@ -859,7 +859,7 @@ func TestConvertEnvinfoURL(t *testing.T) {
 				Kind:   localConfigProvider.INGRESS,
 			},
 			wantURL: URL{
-				TypeMeta:   metav1.TypeMeta{Kind: "url", APIVersion: "odo.dev/v1alpha1"},
+				TypeMeta:   metav1.TypeMeta{Kind: "URL", APIVersion: "odo.dev/v1alpha1"},
```

Comment:
Thoughts on using `URLKind` instead? Or is this intended?

Reply:
Same, it is intended